### ISSUE TITLE
Contribution for usage on Linux

### ISSUE THREAD

**Issue by nestarz** [DjDeveloperr/deno_serial#1]
Hello,

Is there any way to talk to tty serials using Deno ? Can I help contribute if yes ?

Best,

**Comment by DjDeveloperr** [comment]
Yes, as far as I know you'd have to FFI with `libc`. If you could contribute Linux support it would be great! It's only a bit different on macOS so that also becomes possible to support.
